src/kvstore/wal/test/FileBasedWalTest.cpp

```diff
@@ -362,19 +362,23 @@ TEST(FileBasedWal, TTLTest) {
                            folly::stringPrintf("Test string %02d", i)));
     }
     // Wait for the buffer flushed.
-    sleep(1);
+    while (wal->onGoingBuffersNum_ > 0) {
```

Comment:
Do we still need the while scope?  After flusher removed,  onGoingBuffersNum_ will always be zero.

Reply:
Good suggestion.